ably.d.ts

```diff
@@ -3152,6 +3152,16 @@ export declare interface Push {
   deactivate(deregisterCallback?: DeregisterCallback): Promise<void>;
 }
 
+/**
+ * Represents the local device as activated as a target for push notifications.
```

Comment:
```suggestion
 * Contains the device identity token and secret of a device. LocalDevice extends {@link DeviceDetails}.
```

Based on https://sdk.ably.com/builds/ably/specification/main/api-docstrings/

Reply:
addressed in https://github.com/ably/ably-js/pull/2020